service/textmagic/textmagic.go

```diff
@@ -0,0 +1,54 @@
+package textmagic
+
+import (
+	"context"
+	tm "github.com/textmagic/textmagic-rest-go-v2/v2"
+	"strings"
+)
+
+//TMagicService allow you to configure a TextMagic SDK client.
+type TMagicService struct {
+	UserName           string
+	APIKey             string
+	destinations       []string
+	TextMagicAPIClient *tm.APIClient
+}
```

Comment:
```suggestion
// Service allow you to configure a TextMagic SDK client.
type Service struct {
	userName           string
 	apiKey             string
	phoneNumbers       []string
 	client *textMagic.APIClient
}
```

Reply:
Core struct should be called `Service` and none of its fields need to be exported.